build.sbt

```diff
@@ -9,6 +9,7 @@ import scala.xml.transform.{RewriteRule, RuleTransformer}
 // Global settings
 organization in ThisBuild := "org.http4s"
 version      in ThisBuild := scalazCrossBuild("0.15.10-SNAPSHOT", scalazVersion.value)
+relVersion   in ThisBuild := scalazCrossBuild("0.15.9", scalazVersion.value)
```

Comment:
@rossabaker Yeah, I cringed when I added it too.  Any better with commit 5f11b2de ?

/cc @ChristopherDavenport